client/src/components/CKANExampleApp.js

```diff
@@ -29,7 +30,9 @@ class CKANExampleApp extends Component {
    * When the component has mounted, fetch the client schema configuration
    */
   componentDidMount() {
-    const schemaPath = `${window.location.pathname}/schema`;
+    const params = new URLSearchParams(window.location.search || '');
+    const stage = params.has('stage') ? `?stage=${params.get('stage')}` : '';
+    const schemaPath = `${window.location.pathname}/schema${stage}`;
```

Comment:
Fix for #201 . Everything else is cleaning up the loader

Reply:
Would this also need to include things like the Fluent locale or Subsite ID, which may be in the URL params when previewing in the CMS?

Reply:
Possibly? I'm not sure how it's handled when those are omitted? Any advice here?

Reply:
You could pass through all URL params maybe. Otherwise I dunno. This is good for now